### PR TITLE
refactor(profiling): remove deprecated pyddprofile

### PR DESCRIPTION
**Pull request by jd**

### CODE REVIEW

File: releasenotes/notes/profiling-remove-pyddprofile-1a699c195d9d3ab0.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+upgrade:
+  - |
+    The deprecated ``pyddprofile`` wrapper has been removed. Use
+    ``ddtrace-run`` with ``DD_PROFILINE_ENABLE=1`` set instead.
```
Comment (P403n1x87) [inline releasenotes/notes/profiling-remove-pyddprofile-1a699c195d9d3ab0.yaml:5]:
```suggestion
    ``ddtrace-run`` with ``DD_PROFILING_ENABLED=1`` set instead.
```

**Review by P403n1x87 — COMMENTED** [review]
please fix the typo 🙂

File: releasenotes/notes/profiling-remove-pyddprofile-1a699c195d9d3ab0.yaml (hunk 1 above)
Comment (jd) [reply releasenotes/notes/profiling-remove-pyddprofile-1a699c195d9d3ab0.yaml]: fixed

File: releasenotes/notes/profiling-remove-pyddprofile-1a699c195d9d3ab0.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+upgrade:
+  - |
+    The deprecated ``pyddprofile`` wrapper has been removed. Use
+    ``ddtrace-run`` with ``DD_PROFILINE_ENABLED=1`` set instead.
```
Comment (P403n1x87) [inline releasenotes/notes/profiling-remove-pyddprofile-1a699c195d9d3ab0.yaml:5]:
```suggestion
    ``ddtrace-run`` with ``DD_PROFILING_ENABLED=1`` set instead.
```
🔫 too quick 🙂

**Review by Kyle-Verhoog — APPROVED** [review]

**Review by majorgreys — APPROVED** [review]

**Pull request merged**